<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.klindziuk.TriangleTestLab.CheckTriangleTest" tests="23" time="0.086" ignored="0" failures="0" timestamp="22 фев 2017 18:44:33 GMT" skipped="0" hostname="Pk" errors="3">
  <testcase name="checkSides_Test_ValidSide_MAX_ValidSide" time="0.002" classname="com.klindziuk.TriangleTestLab.CheckTriangleTest"/>
  <testcase name="checkSides_Test_ValidSide_NegativeInfinity_ValidSide" time="0.002" classname="com.klindziuk.TriangleTestLab.CheckTriangleTest"/>
  <testcase name="checkSides_Test_Valid_Sides_Maximum" time="0.002" classname="com.klindziuk.TriangleTestLab.CheckTriangleTest"/>
  <testcase name="checkSides_Test_Valid_Sides_Minimum" time="0.002" classname="com.klindziuk.TriangleTestLab.CheckTriangleTest"/>
  <testcase name="constructor_Test_2_TriangleIsLine" time="0.002" classname="com.klindziuk.TriangleTestLab.CheckTriangleTest"/>
  <testcase name="checkSides_Test_ValidSide_PositiveInfinity_ValidSide" time="0.002" classname="com.klindziuk.TriangleTestLab.CheckTriangleTest"/>
  <testcase name="checkSides_Test_ValidSides_Zero" time="0.003" classname="com.klindziuk.TriangleTestLab.CheckTriangleTest"/>
  <testcase name="checkSides_Test_ValidSides_MAX" time="0.002" classname="com.klindziuk.TriangleTestLab.CheckTriangleTest"/>
  <testcase name="checkSides_Test_NaN_ValidSides" time="0.002" classname="com.klindziuk.TriangleTestLab.CheckTriangleTest">
    <error type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.Assert.assertTrue(Assert.java:20)
	at junit.framework.Assert.assertFalse(Assert.java:34)
	at junit.framework.Assert.assertFalse(Assert.java:41)
	at com.klindziuk.TriangleTestLab.CheckTriangleTest.checkSides_Test_NaN_ValidSides(CheckTriangleTest.java:103)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- checkSides_Test_NaN_ValidSides -->
  <testcase name="constructor_Test_3_TriangleIsLine" time="0.002" classname="com.klindziuk.TriangleTestLab.CheckTriangleTest"/>
  <testcase name="checkSides_Test_MAX_ValidSides" time="0.041" classname="com.klindziuk.TriangleTestLab.CheckTriangleTest"/>
  <testcase name="checkSides_Test_ValidSide_Zero_ValidSides" time="0.002" classname="com.klindziuk.TriangleTestLab.CheckTriangleTest"/>
  <testcase name="checkSides_Test__ValidSides_NegativeInfinity" time="0.002" classname="com.klindziuk.TriangleTestLab.CheckTriangleTest"/>
  <testcase name="checkSides_Test_NegativeInfinity_ValidSides" time="0.002" classname="com.klindziuk.TriangleTestLab.CheckTriangleTest"/>
  <testcase name="checkSides_Test_Zero_ValidSides" time="0.002" classname="com.klindziuk.TriangleTestLab.CheckTriangleTest"/>
  <testcase name="checkSides_Test_NegativeSide_ValidSides" time="0.002" classname="com.klindziuk.TriangleTestLab.CheckTriangleTest"/>
  <testcase name="checkSides_Test_ValidSides_NaN" time="0.002" classname="com.klindziuk.TriangleTestLab.CheckTriangleTest">
    <error type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.Assert.assertTrue(Assert.java:20)
	at junit.framework.Assert.assertFalse(Assert.java:34)
	at junit.framework.Assert.assertFalse(Assert.java:41)
	at com.klindziuk.TriangleTestLab.CheckTriangleTest.checkSides_Test_ValidSides_NaN(CheckTriangleTest.java:115)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- checkSides_Test_ValidSides_NaN -->
  <testcase name="checkSides_Test_ValidSide_NegativeSide_ValidSide" time="0.002" classname="com.klindziuk.TriangleTestLab.CheckTriangleTest"/>
  <testcase name="checkSides_Test_PositiveInfinity_ValidSides" time="0.002" classname="com.klindziuk.TriangleTestLab.CheckTriangleTest"/>
  <testcase name="checkSides_Test_ValidSides_NegativeSide" time="0.002" classname="com.klindziuk.TriangleTestLab.CheckTriangleTest"/>
  <testcase name="constructor_Test_1_TriangleIsLine" time="0.002" classname="com.klindziuk.TriangleTestLab.CheckTriangleTest"/>
  <testcase name="checkSides_Test_ValidSide_NaN_ValidSide" time="0.002" classname="com.klindziuk.TriangleTestLab.CheckTriangleTest">
    <error type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.Assert.assertTrue(Assert.java:20)
	at junit.framework.Assert.assertFalse(Assert.java:34)
	at junit.framework.Assert.assertFalse(Assert.java:41)
	at com.klindziuk.TriangleTestLab.CheckTriangleTest.checkSides_Test_ValidSide_NaN_ValidSide(CheckTriangleTest.java:109)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- checkSides_Test_ValidSide_NaN_ValidSide -->
  <testcase name="checkSides_Test_ValidSides_PositiveInfinity" time="0.002" classname="com.klindziuk.TriangleTestLab.CheckTriangleTest"/>
</testsuite> <!-- com.klindziuk.TriangleTestLab.CheckTriangleTest -->
